src/compression/tar/tar-strip-prefix.md

```diff
@@ -7,24 +7,33 @@ the specified path prefix (`bundle/logs`).  Finally, extract the [`tar::Entry`]
 via [`Entry::unpack`].
 
 ```rust,no_run
-# #[macro_use]
-# extern crate error_chain;
 extern crate flate2;
 extern crate tar;
-#
-# error_chain! {
-#     foreign_links {
-#         Io(std::io::Error);
-#         StripPrefixError(::std::path::StripPrefixError);
-#     }
-# }
 
 use std::fs::File;
 use std::path::PathBuf;
 use flate2::read::GzDecoder;
 use tar::Archive;
 
-fn run() -> Result<()> {
+#[derive(Debug)]
```

Comment:
I would hide these new lines by default.  If you put a hash tag in front of them and a space, it would make understanding the core of the recipe easier.